Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM hashicorp/terraform:0.12.29
+
+ENV BUILD_PACKAGES bash curl-dev curl ruby-dev build-base python3
+ENV RUBY_PACKAGES ruby-full
```

Comment:
These are creating cache layers for each `ENV` call. I'd inline these into the `RUN` call below that way the packages and the installation appear in a single cache layer